datafusion/functions-array/src/array_has.rs

```diff
@@ -53,6 +53,16 @@ make_udf_function!(ArrayHasAny,
     array_has_any_udf // internal function name
 );
 
+// TODO: Macro-ify aliases for all functions
+pub fn array_has_aliases() -> Vec<String> {
```

Comment:
the downside with this approach
1.  is that now every time this function is called a new Vec of Strings gets made which is likely close to the same cost as instantiating the struct in the first place 
2. This will will be more expensive as if the struct is ever actually called
3. There are now two Vecs of arrays that need to be kept in sync

Reply:
Agree with the first 2 points.

> There are now two Vecs of arrays that need to be kept in sync

We can replace private aliases.